src/core/smallvec_type.hpp

```diff
@@ -17,6 +17,15 @@
 #include <vector>
 #include <algorithm>
 
+template <typename T>
+inline bool include(std::vector<T>& vec, const T &item)
```

Comment:
doxygen

Reply:
I think i'd prefer if "custom" functions followed OTTD style though - should be CamelCase